arcade/gl/types.py

```diff
@@ -266,7 +271,7 @@ def __init__(
                 f"attributes ({len(self.attributes)})"
             )
 
-        def zip_attrs(formats, attributes):
+        def zip_attrs(formats: list[str], attributes: Sequence[str]):
```

Comment:
List not list, again, does it matter?

Reply:
Good catch, python 3.8 needs it capitalized.